requirements.txt

```diff
@@ -0,0 +1,5 @@
+six>=1.11
+Django>=1.10
```

Comment:
Should we keep supporting the `1.10`?
The mainstream support has ended for it already https://www.djangoproject.com/download/#ft3

Reply:
will update to `1.11`